modules/ROOT/pages/manage-index.adoc

```diff
@@ -3,43 +3,27 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After you install and configure Anypoint Runtime Fabric and deploy Mule applications and API proxies, you might need to perform additional tasks.
-
-* Monitor and View
-
-** View Runtime Fabric status.
-+
-See xref:view-health.adoc[View Runtime Fabric Health Details].
-** Use Anypoint Monitoring and Anypoint Visualizer to monitor applications deployed to Runtime Fabric.
-+
-See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric].
-** Monitor the performance and health of your Runtime Fabric installation.
-
-* Update
-
-** Install rtfctl to manage Runtime Fabric settings.
-+
-See xref:install-rtfctl.adoc[Install rtfctl].
-** Install patches and perform other scheduled maintenance on Runtime Fabric.
-+
-** Replace your Mule license key.
-+
-See xref:replace-license-key.adoc[Replace a Mule License Key].
-
-* Maintain
-
-** Configure proxy settings used by Runtime Fabric outbound connections.
-+
-See xref:manage-proxy-self.adoc[Manage Proxies Used by Runtime Fabric].
-** Manage Runtime Fabric secure properties.
-+
-See xref:manage-secure-properties.adoc[Manage Secure Properties in Runtime Fabric].
-** Enable mutual authentication.
-+
-** Back up Runtime Fabric.
-+
-See xref:manage-backup-restore.adoc[Back up and Restore Runtime Fabric].
-
-* Uninstall Runtime Fabric manually.
-+
-See xref:uninstall-self.adoc[Uninstall Runtime Fabric].
\ No newline at end of file
+To manage Anypoint Runtime Fabric, review the following topics:
+
+* xref:view-health.adoc[Viewing Runtime Fabric Health Details] +
+Use Anypoint Runtime Manager to view the health of your Anypoint Runtime Fabric clusters.
+
+* xref:install-rtfctl.adoc[Using RTFCTL Tool] +
+Use `rtfctl` command line tool to install, configure, upgrade, unistall, troubleshoot, back up and restore Runtime Fabric. Additionally, you can manage deployed Mule apps, protect Mule apps values and install and replace Mule license keys.
+
+* xref:replace-license-key.adoc[Replacing a Mule License Key] +
+Use `rtfctl` command line tool to replace an expired Mule license key.
+
+* xref:downgrade-runtime-fabric.adoc[Downgrading Runtime Fabric Version] +
+Use `rtfctl` command line tool or Helm to downgrade Runtime Fabric versions.
+
+* xref:migrate-helm.adoc[Migrating to Helm Managed Runtime Fabric] +
+Migrate to Helm native for Runtime Fabrics that were installed using `rtfctl`.
+
+* xref:prereq-firewalld-forwarding.adoc[Enabling Forwarding When Using firewalld] +
+Enable forwarding on the docker0 device, if you are using `firewalld` with a Red Hat Enterprise Linux (RHEL) 7.x operating system.
```

Comment:
```suggestion
Enable forwarding on the docker device, if you are using `firewalld` with a Red Hat Enterprise Linux (RHEL) 7.x operating system.
```
Is this a typo? Ignore otherwise.